ssh/DOCS.md

```diff
@@ -249,6 +249,29 @@ based on the following:
 - `MINOR`: Backwards-compatible new features and enhancements.
 - `PATCH`: Backwards-compatible bugfixes and package updates.
 
+## Visual Studio Code Remote - SSH
+
+Some customization is required prior to being able to connect to your Home 
+Assistant install with Visual Studio Code. The following configuration will aid
+in setting up VSCode Remote - SSH:
+
+```yaml
+packages:
+  - gcompat
+  - libstdc++
```

Comment:
It might be worth it to install those by default?

Reply:
Should I then add this to another file? I'm not sure which.

Reply:
You can add it to the list here: https://github.com/hassio-addons/addon-ssh/blob/main/ssh/Dockerfile#L30